identity/identity.go

```diff
@@ -132,6 +133,14 @@ type Identity struct {
 	OrganizationID uuid.NullUUID `json:"organization_id,omitempty"  faker:"-" db:"organization_id"`
 }
 
+func (i *Identity) PageToken() keysetpagination.PageToken {
+	return keysetpagination.StringPageToken(i.ID.String())
+}
+
+func DefaultPageToken() keysetpagination.PageToken {
+	return keysetpagination.StringPageToken(uuid.Nil.String())
```

Comment:
Do we want to support sorting in the future? If so, this should probably be a `MapPageToken`. https://github.com/ory/kratos/blob/master/courier/message.go#L208-L213

https://github.com/ory/x/blob/master/pagination/keysetpagination/paginator.go#L87-L116

Reply:
We can still change that later, right?

Reply:
Not entirely sure. I think it should be possible, but you never know if we break someone then. And if we're going to use the `MapPageToken` then anyway, we might as well do it now, just with one value. I don't think there are any downsides to that. But I haven't worked with it in a while, so feel free to disregard.